agent/handlers/v3/container_metadata_handler.go

```diff
@@ -81,7 +81,7 @@ func GetContainerResponse(containerID string, state dockerstate.TaskEngineState)
 func GetContainerNetworkMetadata(containerID string, state dockerstate.TaskEngineState) ([]containermetadata.Network, error) {
 	dockerContainer, ok := state.ContainerByID(containerID)
 	if !ok {
-		return nil, errors.Errorf("Unable to find container '%s'", containerID)
+		return []containermetadata.Network{}, nil
```

Comment:
This is a change in behavior. 
I think there should be a condition to only return `[]containermetadata.Network{}, nil` only if Known status is <= pulled.
Same below